app/views/admin/apps/_apps_menu.html.haml

```diff
@@ -0,0 +1,11 @@
+%h2= t('apps.admin.title')
+%hr
+
+%ul.nav.nav-pills.nav-stacked.apps-nav
+  - @apps.each do |app|
+    = nav_tab(:id, :app_id, app.subdomain) do
+      = link_to(admin_app_path(app)) do
+        %h5{style: 'word-wrap: break-word;'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace